### DISCUSSION TITLE
Website to teach Traffic rules for Egyptians based on the German System.

### DISCUSSION THREAD

**Discussion by NohaelKa** [markusbegerow/Wp-Adv-Quiz#37] [Show and tell]
It is the only site of its kind to teach Traffic law through question answer format in Egypt. In addition it gives additional information based on the German test that are not exisitng in Egyptian law, so it serves as an improved proposal.

Website: https://ta3limsewa2a.elkmh.com/
Driving test: https://ta3limsewa2a.elkmh.com/%d8%a7%d9%84%d8%a5%d9%85%d8%aa%d8%ad%d8%a7%d9%86/

**Comment by markusbegerow** [comment]
It looks awesome @NohaelKa!

![image](https://github.com/markusbegerow/Wp-Adv-Quiz/assets/44146279/d599fa0a-41df-4d59-bc64-34b7de059438)
